.ci/setup.sh

```diff
@@ -45,6 +45,9 @@ bash -f ${cidir}/install_shim.sh
 echo "Install proxy"
 bash -f ${cidir}/install_proxy.sh
 
+echo "Install ksm throttler"
+bash -f ${cidir}/install_ksm_throttler.sh
```

Comment:
with the chmod to the `install_ksm_throttler.sh` in this commit, do you still need the `bash -f` ?